From e9d21c7290f07582f776832a9ba9af207e8a9486 Mon Sep 17 00:00:00 2001
From: lichenggang <lichenggang@deepin.org>
Date: Fri, 28 Nov 2025 17:49:01 +0800
Subject: [PATCH] skip tests fpi-device

---
 tests/meson.build | 1 -
 1 file changed, 1 deletion(-)

diff --git a/tests/meson.build b/tests/meson.build
index f68ed40..19691ce 100644
--- a/tests/meson.build
+++ b/tests/meson.build
@@ -245,7 +245,6 @@ test_utils = static_library('fprint-test-utils',
     install: false)
 
 unit_tests = [
-    'fpi-device',
     'fpi-ssm',
     'fpi-assembling',
 ]
-- 
2.47.2

