--- libfprint-1.94.7.orig/libfprint/drivers/uru4000.c
+++ libfprint-1.94.7/libfprint/drivers/uru4000.c
@@ -1455,6 +1455,18 @@ fpi_device_uru4000_init (FpiDeviceUru400
 }
 
 static void
+suspend (FpDevice *dev)
+{
+  fpi_device_suspend_complete (dev, fpi_device_error_new (FP_DEVICE_ERROR_NOT_SUPPORTED));
+}
+
+static void
+resume (FpDevice *dev)
+{
+  fpi_device_resume_complete (dev, NULL);
+}
+
+static void
 fpi_device_uru4000_class_init (FpiDeviceUru4000Class *klass)
 {
   FpDeviceClass *dev_class = FP_DEVICE_CLASS (klass);
@@ -1474,4 +1486,7 @@ fpi_device_uru4000_class_init (FpiDevice
 
   img_class->img_width = IMAGE_WIDTH;
   img_class->img_height = IMAGE_HEIGHT;
+
+  dev_class->suspend = suspend;
+  dev_class->resume = resume;
 }
